charts/sn-platform/templates/prometheus/prometheus-configmap.yaml

```diff
@@ -79,10 +79,14 @@ data:
       - source_labels: [__meta_kubernetes_pod_label_job_name]
         action: drop
         regex: ".+"
-      # ignore prometheus,grafana,toolset
+      # Only discover pod in sn-platform target namespace
+      - source_labels: [__meta_kubernetes_namespace]
+        action: keep
+        regex: "{{ template "pulsar.namespace" . }}"
```

Comment:
We can control the discover namespace scopes with the namespaces field directly.
https://github.com/streamnative/charts/blob/43d7cb585da83fc5fd39c396ec96f0201a73923e/charts/sn-platform/templates/prometheus/prometheus-configmap.yaml#L62-L64

Reply:
@ciiiii Tested, apply the suggestions in https://github.com/streamnative/charts/pull/792/commits/164c72790d4ba89bd98a0e22fa5b9bb7e15f8d0a